code/game/machinery/computer/cloning.dm

```diff
@@ -270,22 +272,22 @@
 		if("clone")
 			var/cost = selected_pod.get_cloning_cost(scanner.last_scan, desired_data)
 			if(selected_pod.biomass < cost[BIOMASS_COST] || (selected_pod.reagents.get_reagent_amount("sanguine_reagent") < cost[SANGUINE_COST]) || selected_pod.reagents.get_reagent_amount("osseous_reagent") < cost[OSSEOUS_COST])
-				feedback = list("text" = "The cloning operation is too expensive!", "color" = "bad")
+				feedback = list("text" = "The cloning operation is too expensive!", "color" = "bad", "scan_succeeded" = FALSE)
 			else
 				selected_pod.start_cloning(scanner.last_scan, desired_data)
 				scanner?.update_scan_status()
-				feedback = list("text" = "Beginning cloning operation...", "color" = "good")
+				feedback = list("text" = "Beginning cloning operation...", "color" = "good", "scan_succeeded" = TRUE)
 			return TRUE
 		if("scan")
 			if(!COOLDOWN_FINISHED(src, scancooldown))
-				feedback = list("text" = "The scanning array is still calibrating! Please wait...", "color" = "good")
+				feedback = list("text" = "The scanning array is still calibrating! Please wait...", "color" = "good", "scan_succeeded" = FALSE)
```

Comment:
```suggestion
				feedback = list("text" = "The scanning array is still calibrating! Please wait...", "color" = "average", "scan_succeeded" = FALSE)
```
I snowflaked this in because otherwise my PR wouldn't be too atomic